packages/webdriver/src/monad.js

```diff
@@ -105,6 +105,8 @@ export default function WebDriver (options, modifier, propertiesObject = {}) {
             Promise.resolve(result).then((res) => {
                 log.info('RESULT', res)
                 this.emit('result', { name, result: res })
+            }).catch(err => {
```

Comment:
adding catch here is not affecting error handling at all, it doesn't matter if sync is on or off.